charts/generic-service/postgres-restore.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -e
+
+PRISON_API_BASE_URL=https://api-preprod.prison.service.justice.gov.uk
```

Comment:
hard coded Prison API for pre-production since the job is only designed to copy the database from production to pre-production.